packages/markdown/components/HTMLBlock.jsx

```diff
@@ -0,0 +1,19 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+
+class HTMLBlock extends React.Component {
+  render() {
+    const { html } = this.props;
+    return <div className="rdmd-html" dangerouslySetInnerHTML={{ __html: html }} />;
```

Comment:
What a name!

Reply:
Is this exposed to the client with that directive?

Reply:
Nope. Internal only